include/RAJA/util/View.hpp

```diff
@@ -182,10 +184,16 @@ struct MultiView
   RAJA_INLINE constexpr MultiView(
```

Comment:
You should be able to make a set_data that uses the same kind of SFINAE this uses to allow set_data(nc_pointer_type) to work for a const data MultiView.